low-level/sspl-ll.spec

```diff
@@ -133,7 +133,7 @@ rm -f /etc/dbus-1/system.d/sspl-ll_dbus_policy.conf
 rm -f /usr/bin/resource_health_view /usr/bin/sspl_bundle_generate /usr/bin/manifest_support_bundle
 
 %files
-%defattr(-,sspl-ll,root,-)
+%defattr(-,root,root,-)
```

Comment:
this would change file / directories permissions for use by root user alone. sspl-ll user, with which sspl_ll_d runs, will have issues writing to those dirs/files again ? How sanity passing with these failures ?

Reply:
> Mariyappan - This is handled in post_install where we provide ownership to sspl-ll after sspl-ll user creation.

Understood, fine then.